webapp/src/services/near-nep21-util.js

```diff
@@ -112,11 +120,16 @@ export async function swapFromIn( token1, token2 ) {
     }
     else {
       // NEP-21 to Native
-      const price = await window.contract.swap_reserve_to_near_exact_in( {
-        token: token1.address,
-        tokens_paid: amount1,
-        min_near: amount2 });
-      return price;
+      if(token2.type === "Native token") {
```

Comment:
you don't need to create a new level of `if` statement, instead, please combine it in the upper level (`else if ()`)